### PR TITLE
[tw] Add content/zh-tw/zero-trust-architecture.md

### LINKED ISSUE #2187: [tw] Localize `Zero Trust Architecture` for Traditional Chinese
### Please describe why this term needs to be updated

- Localize `Zero Trust Architecture` for Traditional Chinese
- https://glossary.cncf.io/zero-trust-architecture/

### PR DESCRIPTION
**Pull request by kaiyee0**
### Describe your changes

Origin content from https://glossary.cncf.io/zero-trust-architecture/

### Related issue number or link (ex: `resolves #issue-number`)
resolves #2187

### Checklist before opening this PR (put `x` in the checkboxes)
- [x] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [x] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

**Comment by jihoon-seo** [comment]
To Team Taiwan:
currently auto-labeler is not working for PRs that are set `dev-tw` as the target branch.
To make this workflow work, please update the `dev-tw` branch, by opening a PR using this link: https://github.com/cncf/glossary/compare/dev-tw...main

File: content/zh-tw/zero-trust-architecture.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+---
+title: 零信任架構
+status: Completed
+category: 概念
+tags: ["安全性", "", ""]
+---
+
+## 是什麼 {#what-it-is}
```
Comment (pichuang) [inline content/zh-tw/zero-trust-architecture.md:8]: [MD022 - Headings should be surrounded by blank lines](https://github.com/DavidAnson/markdownlint/blob/v0.29.0/doc/md022.md)

File: content/zh-tw/zero-trust-architecture.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+零信任架構是一種完全移除信任的 IT 系統設計與實作方法。
+其核心的原則是「永不信任，始終驗證」，系統中的裝置或系統本身在與其他組件溝通前，都必須先驗證自己的身份。
+在許多企業網路中，由於系統與裝置都在企業網路的邊界內，因此他們大多時候可以自由溝通。
+然而，零信任架構卻採取了相反的做法：即使在網路邊界內，系統中的組件在進行任何溝通前，都必須先通過驗證。
```
Comment (pichuang) [inline content/zh-tw/zero-trust-architecture.md:12]:
```suggestion
然而，零信任架構卻採取了相反的做法：即使在網路邊界內，系統中的元件在進行任何溝通前，都必須先通過驗證。
```

components 台灣常用字應為`元件`非`組件`

File: content/zh-tw/zero-trust-architecture.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
+---
+title: 零信任架構
+status: Completed
+category: 概念
+tags: ["安全性", "", ""]
+---
+
+## 是什麼 {#what-it-is}
+零信任架構是一種完全移除信任的 IT 系統設計與實作方法。
+其核心的原則是「永不信任，始終驗證」，系統中的裝置或系統本身在與其他組件溝通前，都必須先驗證自己的身份。
```
Comment (pichuang) [inline content/zh-tw/zero-trust-architecture.md:10]:
```suggestion
其核心的原則是「永不信任，始終驗證」，系統中的裝置或系統本身在與其他元件溝通前，都必須先驗證自己的身份。
```

File: content/zh-tw/zero-trust-architecture.md (hunk 1 above)
Comment (kaiyee0) [reply content/zh-tw/zero-trust-architecture.md:8]: fixed

File: content/zh-tw/zero-trust-architecture.md (hunk 2 above)
Comment (kaiyee0) [reply content/zh-tw/zero-trust-architecture.md:12]: fixed

File: content/zh-tw/zero-trust-architecture.md (hunk 3 above)
Comment (kaiyee0) [reply content/zh-tw/zero-trust-architecture.md:10]: fixed

**Review by ydFu — APPROVED** [review]
LGTM

**Review by pichuang — APPROVED** [review]
LGTM

**Pull request merged**